### PR TITLE
removed /client route, it was not working since a long time

### PR DESCRIPTION
**Pull request by fcaps**
it was also not working before my change to the github-downlaod stuff. 
also..  don't get the use of a download link redirected via the server (maybe tracking?), but hey, who knows

### CODE REVIEW

**Review by Brutus5000 — CHANGES_REQUESTED** [review]
The corresponding outdated a-element in index.pug should be removed too

**Comment by fcaps** [comment]
> The corresponding outdated a-element in index.pug should be removed too

good catch, was not aware of it, maybe bc. it was already commented out xD

File: express.js [hunk 1]
```diff
@@ -215,11 +215,6 @@ app.get('/account/checkUsername', require('./routes/views/checkUsername'));
 app.get('/password_resetted', require(routes + 'account/get/requestPasswordReset'));
 app.get('/report_submitted', require(routes + 'account/get/report'));
 // Download Client
```
Comment (Sheikah45) [inline express.js]: This comment can be removed as well correct?

Comment (fcaps) [reply express.js]: done

Comment (fcaps) [reply express.js]: we need to cleanup the express.js so bady, maybe if i got the energy for it.

**Review by Sheikah45 — APPROVED** [review]

**Pull request merged**